kahuna/public/js/components/gr-panels/gr-panels.css

```diff
@@ -51,3 +48,7 @@
     display: block;
     margin-top: 36px;
 }
+
+.gr-panel-height {
+    height: calc(100vh - 86px); /* 50px search bar, 36px toolbar */
```

Comment:
I think I was overcomplicating  / abstracting things - so it was obviously broken.